lib/sm/api/folders.rb

```diff
@@ -11,19 +11,16 @@ def get_folders
         Common::Collection.new(Folder, json)
       end
 
-      # get_folder: Retrieves a folder by its id.
```

Comment:
These method names are sufficiently descriptive enough that comments are not needed.